ethereum_history_api/circuits/lib/src/fixtures/paris/usdc/storage_proof.nr

```diff
@@ -0,0 +1,28 @@
+    ],
+    depth: 7
+  }
+
```

Comment:
And here

Reply:
I understand that all those formatting shenanigans will be fixed as soon as we can just run prettier for noir, but for now we can not, so let's try to generate beautiful code if it's not too much work. Here it's just removing a couple of tabs and new lines

Reply:
removed